interns-react-weekly-assignments/src/components/Home/Keshav/Week4/TicTacToe/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { TicTacToeWrapper } from './styles';
+
+const TicTacToe = () => {
+  const [ticTacToeBoard, setTicTacToeBoard] = useState<(number | string)[][]>([
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
+  ]);
+  const [player, setPlayer] = useState<number>(1);
+  const [winner, setWinner] = useState<number | null>(null);
+
+  const handleMove = (row: number, column: number) => {
+    if (ticTacToeBoard[row][column] !== ' ' || winner !== null) return;
+
+    const newBoard = [...ticTacToeBoard];
+    newBoard[row][column] = player === 1 ? 'X' : 'O';
+
+    if (checkWinner(newBoard, player === 1 ? 'X' : 'O')) {
+      setWinner(player);
+      return;
+    }
+
+    setPlayer(player === 1 ? 2 : 1);
+    setTicTacToeBoard(newBoard);
+  };
+
+  const checkWinner = (board: (number | string)[][], symbol: string): boolean => {
```

Comment:
since symbol is a JS datatype I would suggest choosing a different name for this argument to avoid confusion. I think you dont need that argument
you can do something like 
if (board[i][0] === board[i][1]  && board[i][1] === board[i][2]) return board[i][0]
but here you might have to handle the empty case